### PR TITLE
UI card

### PR DESCRIPTION
**Pull request by pldespaigne**
multiple ui fixes related to #5625

### CODE REVIEW

File: styles/components/mat-tabs.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  $warn: map-get($theme, warn);
+  $background: map-get($theme, background);
+  $foreground: map-get($theme, foreground);
+
+  .mat-tab-label:hover {
+    .mat-icon {
+      path, g, polygon {
+        fill: mat-color($foreground, base);
+      }
+    }
```
Comment (Mussuranae) [inline styles/components/mat-tabs.scss:13]: You could put it directly on the mat-icon file no ?

Comment (pldespaigne) [reply styles/components/mat-tabs.scss:13]: I could but that would put `.mat-tab-label` in the mat-icon file.
Imo both solution are ok and none of them are perfect ^^

Comment (Mussuranae) [reply styles/components/mat-tabs.scss:13]: And we want this behavior only when the icon is inside a tab ?

**Review by Mussuranae — APPROVED** [review]

File: styles/components/mat-tabs.scss (hunk 1 above)
Comment (Mussuranae) [reply styles/components/mat-tabs.scss:13]: If yes, so that's okay, if no and we want it for all icons, you can move it to the icon file :)

**Pull request merged**